include/graphblas/reference/vector.hpp

```diff
@@ -231,6 +231,7 @@ namespace grb {
 		friend uintptr_t getID( const Vector< InputType, reference, Coords > & );
 
 		friend class PinnedVector< D, reference >;
+		friend class PinnedVector< D, nonblocking >;
 
```

Comment:
Remove this new line? (Or add one before the nonblocking friend-- code style has no clear preference here, so the only thing's to be consistent)